modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/PageRenderer/PaginatedVariant.es.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import React from 'react';
+
+import {Pagination} from '../Pagination.es';
+import {PaginationControls} from '../PaginationControls.es';
+import * as DefaultVariant from './DefaultVariant.es';
+
+export const Container = ({
```

Comment:
This is the variation of Paginated, apparently, I don't see it in Forms anymore, I'm wondering if we can remove the support from this variation but I'm not sure if it is still here for backward compatibility reasons 🤷‍♂️. cc @brunobasto

Reply:
We can ask the Forms team. If they ditched it we might be able to remove it. Although we need to think about legacy data. Did they write or will they write an upgrade path for forms in the database that were using this kind of pagination?

Reply:
This is a great question!